lms/static/sass/_build-discussion.scss

```diff
@@ -0,0 +1,16 @@
+// From _build-lms-v1.scss
```

Comment:
Fair enough. Could you just move the files into the discussion subdirectory, then. That will keep them more clearly separated. Thanks.